MMM-Afvalwijzer.js

```diff
@@ -92,6 +93,7 @@ Module.register('MMM-Afvalwijzer', {
             case 'PLASTICPLUS':
                 color = "#e96c29";
                 break;
+            case 'papier':
```

Comment:
can be removed as it only uses 'papier and karton'

Reply:
@gertperdZA 
My data does show 'papier' as an option. The paper bins didn't turn up as blue, therefore I have added it.
<img width="311" alt="image" src="https://github.com/user-attachments/assets/6f1fd125-ccf0-492f-8229-84b35caa852d">